### ISSUE TITLE
Cropper no longer starts minimized.

### ISSUE THREAD

**Issue by patricknelson** [brhinescot/Cropper#20]
Beginning with the latest version (1.11.0), cropper always starts up with the form displayed. I set this up to boot with Windows (and use it very frequently) so I don't need it to display on each login/reboot. I happened to notice this today after setting up a new computer and the blue form shows each time the application is launched. Version 1.10.0 is not affected, so I've downgraded to that.

Bug? Or, maybe feature request: Add the ability to configure this as an option (some people may actually prefer this). I'd suggest making the default value for this to be hidden on start up, carrying over for legacy reasons and, most likely, the most common use case 😄

**Comment by tomasz1986** [comment]
I have exactly the same case here. Cropper is set to start automatically on boot. The older versions used to start hidden in tray, but now the program window is displayed on screen. If possible, please either restore the old behaviour or just add an option to start it minimized.

**Comment by brhinescot** [comment]
This must be a bug introduced in the previous version. Looking in to it.

**Comment by brhinescot** [comment]
Yes, Cropper previously loaded the last state, hidden or not, from the previous run's saved configuration, so if it stopped hidden it would restart hidden. That line was mistakenly removed in the last build during some restructuring of the configuration UI. I'll get a build out with the fix. Thanks!

**Comment by eNdEmiOn** [comment]
I have the same issue but it's 2024 now so this never got fixed I guess as we're still on 1.11.0 (I only recently started using a screen (1440p) which required an update Cropper wise).

**Comment by patricknelson** [comment]
Wow, blast from the past.

I think not long after this I started using ShareX. There are lots of good alternatives these days, probably worth looking into those since it appears Cropper is no longer maintained (unless you wanna fork and update it yourself).

**Comment by eNdEmiOn** [comment]
I actually did look for alternatives but you end up with tools that feature loads of stuff you don't need or they can't actually use print screen button integration.

I've been trying out that ShareX tool even though it has loads of features I don't need (but I guess it could replace another tool too, that wasn't working well anymore) and is kinda hard to set up as a replacement for Cropper as the settings are a bit "all over the place". I mean why are the file naming settings in a completely different section from Paths settings. Had to consult the interwebs to figure that out and a bunch of other basic things aren't all that clear.
